samples/bluetooth/peripheral_with_multiple_identities/README.rst

```diff
@@ -0,0 +1,190 @@
+#. Utilise a single physical device to emulate multiple Bluetooth advertisers.
+#. Each advertiser is assigned a unique identity and a corresponding random static address, ensuring they can be distinctly identified.
+#. Advertisers are restarted upon disconnection.
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following Kconfig options:
+
+.. _CONFIG_BT_ID_MAX:
+
+CONFIG_BT_ID_MAX
+   This configuration defines how many identities will be used.
+
+.. _CONFIG_BT_MAX_CONN:
+
+CONFIG_BT_MAX_CONN
+   This configuration defines how many connections which can be established.
+   The sample expects this configuration to be set to the same value as **CONFIG_BT_ID_MAX**.
+
+.. _CONFIG_BT_EXT_ADV_MAX_ADV_SET:
+
+CONFIG_BT_EXT_ADV_MAX_ADV_SET
+   This configuration defines how the number of advertising sets that are available.
+   The sample expects this configuration to be set to the same value as **CONFIG_BT_ID_MAX**.
+
+.. _CONFIG_BT_BUF_ACL_RX_COUNT:
+
+CONFIG_BT_BUF_ACL_RX_COUNT
+   This configuration defines the number of incomming ACL data buffers sent from controller to the host.
+   This value needs to be greater than **CONFIG_BT_MAX_CONN**.
+
+.. _CONFIG_BT_DEVICE_NAME:
+
+CONFIG_BT_DEVICE_NAME
+   This configuration defines the base name given to each advertiser.
+
+How it works
+************
+
+Setting up the advertisers
+==========================
+
+The sample application will setup and start as many advertisers as specified by **CONFIG_BT_EXT_ADV_MAX_ADV_SET**.
+The :c:func:`setup_advertiser()` configures and initialises a connectable advertiser with a unique name and identity:
+
+#. Identities are created with the API :c:func:`bt_id_create()`.
+#. To assign a given identity to an advertiser, the field :c:member:`bt_le_adv_param.id` is set.
+#. A unique name is then generated for each advertiser consiting of **CONFIG_BT_DEVICE_NAME** value followed by the identity.
+#. The APIs :c:func:`bt_le_ext_adv_create()` and :c:func:`bt_le_ext_adv_set_data()` will create the advertising set and set its advertising data, respectively.
+
+Each advertiser is setup as a connectable legacy advertiser but the extended advertising version of the API is used as it supports multiple advertising instances and sets. 
+
+A work item is then initialized for each advertiser, in order to avoid race conditions and ensure that tasks are executed independently.
+Each work items handler function is set to :c:func:`start_connectable_advertiser()` which will start the advertiser associated with the work item.
+This is done by calling :c:func:`bt_le_ext_adv_start()`. Note: By default this function will not attempt to resume connectable advertising after a connection is established. 
+Each advertisers work item is then submitted to the workqueue to handle the starting of each advertiser asynchronously.
+The sample will then continue executing asynchronously using callbacks to define its behavior.
```

Comment:
```suggestion
A work item is initialized for each advertiser to avoid race conditions and ensure that tasks are executed independently.
Use the :c:func:`start_connectable_advertiser` function to start the advertiser associated with the work item and call :c:func:`bt_le_ext_adv_start`.
By default, this function does not attempt to resume connectable advertising after a connection is established. 
Each advertiser's work item is submitted to the work queue to handle the starting of each advertiser asynchronously.
The sample continues to execute asynchronously using callbacks to define its behavior.
```
I may have missed the point here. Should these be steps as well?